3rdparty/invlib/src/invlib/map.cpp

```diff
@@ -63,6 +63,11 @@ ::cost_function(const VectorType &x,
         if (robust) {
             return std::numeric_limits<RealType>::max();
         }
+        std::throw_with_nested(
```

Comment:
IMO nesting the error once more is not necessary because that's already done in the `evaluate(...)` call.

Reply:
Good point. I changed it to just rethrow the exception.